source/RunClient.py

```diff
@@ -18,14 +17,20 @@ def RunClient():
     connection.DoConnect((host, int(port)))
     clientState = ClientState(ruleset)
     gameControl = Controller(clientState)
-    handView = HandView(gameControl)
-    tableView = TableView()
-    while 1:
+    playername = gameControl.getName()
+    createDisplay = CreateDisplay(playername)
```

Comment:
I don't love this name - initialization makes sense to call create, but I don't think of it as recreating each render. That said, its not something worth holding up the commit over.